tests/network-ovn

```diff
@@ -2091,6 +2091,14 @@ ovn_leases_tests() {
 
         lxc delete -f c1
 
+        if hasNeededAPIExtension list_ovn_uplink_allocations; then
+            echo "==> Check OVN external addresses show on lxc network list-allocations"
+            ovn_ext_ipv4="$(lxc network get ovn-virtual-network volatile.network.ipv4.address)"
+            ovn_ext_ipv6="$(lxc network get ovn-virtual-network volatile.network.ipv6.address)"
+            lxc network list-allocations --format csv | grep -e "/1.0/networks/ovn-virtual-network,${ovn_ext_ipv4}"
+            lxc network list-allocations --format csv | grep -e "/1.0/networks/ovn-virtual-network,${ovn_ext_ipv6}"
```

Comment:
```suggestion
            lxc network list-allocations --format csv | grep -F "/1.0/networks/ovn-virtual-network,${ovn_ext_ipv4}"
            lxc network list-allocations --format csv | grep -F "/1.0/networks/ovn-virtual-network,${ovn_ext_ipv6}"
```

Sorry for not mentioning this on the first review pass. Using fixed string should be safer as IPv4 contain `.` which matches everything/anything as you know.

Reply:
No problem at all!
Good catch btw, thanks